index.html

```diff
@@ -10,22 +11,14 @@
 </head>
 
 <body>
-    <div class="hero_content">
-        <header class="header">
```

Comment:
Can you change it as it was since we are going to set the navmenu inside the header?